otel-integration/k8s-helm/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: otel-integration
 description: OpenTelemetry Integration
-version: 0.0.203
+version: 0.0.204
```

Comment:
Should maybe add a lint to always expect a version bump unless you explicitly add a label to skip-version-bump

Reply:
Good idea, @nicolastakashi. By chance, have you seen something like this in any other repository that we could use as a base for our own solution?